### PR TITLE
NAV LAUNCH: code tidy and add altitude where leave the launch

### PR DESCRIPTION
**Pull request by giacomo892**
The model will leave launch mode when the target altitude is reached (if  launch timeout doesn't kick before) and switch back to the current flight mode.
Also tidied a bit conditions check

### CODE REVIEW

File: src/main/navigation/navigation_fw_launch.c [hunk 1]
```diff
@@ -137,6 +137,30 @@ static void applyFixedWingLaunchIdleLogic(void)
...
+{
+    return timeSinceLaunchMs >= navConfig()->fw.launch_timeout;
+}
+
+static inline bool isLaunchModeFinishedByPilot()
+{
+    return (ABS(rcCommand[ROLL]) > rcControlsConfig()->pos_hold_deadband) || (ABS(rcCommand[PITCH]) > rcControlsConfig()->pos_hold_deadband);
+}
+
+static inline bool checkFixedWingLaunchState(float timeSinceLaunchMs)
```
Comment (digitalentity) [inline src/main/navigation/navigation_fw_launch.c]: The name is confusing. Possibly `isFixedWingLaunchCompleted` is a better name?

**Review by digitalentity — COMMENTED** [review]
Otherwise than a minor naming clarity issue - I'm good with this

File: src/main/navigation/navigation_fw_launch.c (hunk 1 above)
Comment (giacomo892) [reply src/main/navigation/navigation_fw_launch.c]: Followed your suggestion and updated Cli documentation too.

**Comment by digitalentity** [comment]
@giacomo892 cool, thanks!

**Pull request merged**